neurograph.py

```diff
@@ -0,0 +1,170 @@
+import networkx as nx
+from networkx.algorithms import bipartite
+
+#input: DAG g as an adjacent list representation 
+# b is in a's adj list if a --> b
+#construct transitive closure g' using DFS
+#store a path dict (trans_path), where key = vertice v, value = a dict maps reachable nodes u to a path of v to u
+#store a graph dict trans_dict (transitive closure of g, adjacent list representation)
+# for node v, return a dict maps all reachable nodes u to a path of v to u
+
+def helper(g, trans_path, v):
```

Comment:
Maybe give this a more descriptive name?

Reply:
Resolved